lib/src/linking.rs

```diff
@@ -150,6 +150,12 @@ fn link_legacy_aliases(linker: &mut Linker<WasmCtx>) -> Result<(), Error> {
         "env",
         "xqd_req_downstream_client_ip_addr",
     )?;
+    linker.alias(
+        req,
+        "downstream_client_h2_fingerprint",
+        "env",
+        "xqd_req_downstream_client_h2_fingerprint",
+    )?;
```

Comment:
New features should be binding against the official name, we shouldn't be introducing new legacy names
```suggestion
```

Reply:
@acfoltzer do you happen to know what the new official name is?

The name in the Rust SDK is `downstream_client_h2_fingerprint` -- https://docs.rs/fastly/latest/src/fastly/http/request/handle.rs.html#890

Reply:
The official name is what the witx translates to, so `fastly_http_req::downstream_client_h2_fingerprint` (or however you would spell it in your local environment). The `xqd_*` names are leftover from before we used witx, and are kept around only for compatibility with modules compiled against very old SDKs.

That's all to say that accepting the suggestion to delete this particular change will take care of this entirely 🙂